cmd/dashboard.go

```diff
@@ -165,7 +165,7 @@ dapr dashboard -k -p 0
 			}()
 
 			// url for dashboard after port forwarding.
-			var webURL string = net.JoinHostPort(dashboardHost, fmt.Sprint(portForward.LocalPort))
+			var webURL string = fmt.Sprintf("%s%s", "http://", net.JoinHostPort(dashboardHost, fmt.Sprint(portForward.LocalPort)))
```

Comment:
```suggestion
			var webURL string = fmt.Sprintf("http://%s", net.JoinHostPort(dashboardHost, fmt.Sprint(portForward.LocalPort)))
```

Reply:
On another thought, is there any utility for using `net.JoinHostPort`?

Reply:
@mukundansundar ?

Reply:
I have not found a utility as yet ... IMO that can be a refactor that can be done separately ...